projects/cashmere/src/lib/tabs/tab.component.ts

```diff
@@ -8,11 +9,17 @@ import {Component, Input} from '@angular/core';
     styles: []
 })
 export class TabComponent {
-    /** Text to be displayed on the tab */
+    /** Title of the tab (HTML markup supported) */
     @Input() tabTitle: string = '';
     /** Router path that the tab routes to. If one tab uses the routerLink in a tab set, all must use the router link.
      * Can be specified as '/path/2' or ['path', '2']
      */
     @Input() routerLink: any[] | string;
     _active: boolean = false;
+
+    constructor(private sanitizer: DomSanitizer) {}
+
+    _safeHtmlTitle(): SafeHtml {
+        return this.sanitizer.bypassSecurityTrustHtml(this.tabTitle);
```

Comment:
This was necessary to keep the standard HTML sanitizer from stripping out a lot of standard markup like inline styles, etc.  Would like to verify with those of you who know more about security than me that this isn't introducing a significant security risk.

Reply:
There is some risk in doing this such as:

```
<hc-tab [tabTitle]="someUserProvidedValue">
```

In this case, the user could provide whatever HTML they want to and it would be injected here without being sanitized.  This could pose a security risk.